analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx

```diff
@@ -37,7 +37,8 @@ class EssenceFont extends Analyzer {
   cancelled_ef: number = 0;
   hasUpwelling: boolean = false;
   cancelDelta: number = 100;
-  last_ef: number = 0;
+  last_ef_time: number = 0;
+  last_ef: any = null;
```

Comment:
BAD
NEVER DO ANY. @Pewtro WILL MURDER YOU IF YOU DO
You should do CastEvent | null

Reply:
Fixed